webclient/impls/core/src/test/config/javascript/context.js

```diff
@@ -26,3 +26,10 @@ var requireCfg = {
 };
 
 var KARMA_RUN = true;
+
```

Comment:
Required so that `underscore.js` loads registering as a global variable. Similar technique to that used by CDF JS tests.

RequireJS is restored by the `karma.main.js` file.